sources/sql_database/pg_replication/helpers.py

```diff
@@ -0,0 +1,528 @@
+from typing import Optional, Dict, Iterator, Union, List, Tuple, Sequence, Any
+from dlt.common.pendulum import pendulum
+from copy import deepcopy
+from dataclasses import dataclass, field
+
+import psycopg2
```

Comment:
there will be people using `sql_database` that are not using postgres replication. we are not surfacing this into top level init? anyway... I think we may land it in separate verified source (if we do not share anything with sql alchemy source)

Reply:
Separate source makes sense to me. Only thing is that the `table_snapshot` resource for initial loading uses the `sql_table` resource under the hood. How should we handle one verified source depending on another?